emailTemplates/morning-unclaimed.html

```diff
@@ -0,0 +1,844 @@
+<!doctype html>
```

Comment:
Template provided by @thebitguru over slack message. Moved into repo so that they can be tracked with git and updated in database as part of deploy workflow